orc8r/gateway/go/services/magmad/main.go

```diff
@@ -65,6 +65,8 @@ var (
 )
 
 func main() {
+	err := service.StartMagmadServer()
+	fmt.Print(err)
```

Comment:
This seems like leftover debug code.

Reply:
Weird, I have no recollection of writing and committing this code. I removed it anyway.